scripts/memory_logger.py

```diff
@@ -813,7 +813,11 @@ def buildGraph(self):
 
             # Populate the graph
             for records in value_list:
-                tmp_memory.append(decimal.Decimal(records[1]) / self.multiples)
+                if decimal.Decimal(records[1]) == 0:
```

Comment:
self.multiples is created by a bitwise method. Its always greater than 0:
```python
        # Determine the scale of the graph
        suffixes = ["Terabytes", "Gigabytes", "Megabytes", "Kilobytes", "Bytes"]
        multiplier = 1 << 40;
        index = 0
        while largest_memory[-1] < multiplier and multiplier >= 1:
            multiplier = multiplier >> 10
            index = index + 1
        self.multiples = multiplier
```

Its the possible file being read in (records[]), with 0 bytes as the recorded usage that is the culprit.